cypress/e2e/consent.cy.ts

```diff
@@ -67,12 +63,12 @@ const expectAdFree = (reasons: AdFreeCookieReasons[]) => {
 };
 
 describe('tcfv2 consent', () => {
-    beforeEach(() => {
-        cy.clearCookies();
-        cy.clearLocalStorage();
-    });
+	beforeEach(() => {
+		cy.clearCookies();
+		cy.clearLocalStorage();
+	});
 
-	pages.forEach(({ path, adTest }) => {
```

Comment:
Nothing has functionally changed, we still test one page: the home page. Before, there was only one front that existed and we set `pages` to be this front. Now, we use the first item in the array of fronts.

Reply:
We only want to test one page and this suite of tests take a long time and we would not get much more value from testing extra pages.